yarn-project/circuits.js/src/cbind/circuits.gen.ts

```diff
@@ -828,6 +837,413 @@ export function fromPreviousKernelData(o: PreviousKernelData): MsgpackPreviousKe
   };
 }
 
+export interface MsgpackCallContext {
```

Comment:
Probably should've asked this in the previous PR, but can we **not** export these...?

Reply:
+1 for not exporting